edx_sga/sga.py

```diff
@@ -253,18 +253,18 @@ def student_state(self):
             uploaded = None
 
         if self.annotated_sha1:
-            annotated = {"filename": self.annotated_filename}
+            annotated = {"filename": unicode(self.annotated_filename)}
```

Comment:
Can you use the equivalent function from `six` instead, here and below?

Reply:
@noisecapella Any thoughts on https://pythonhosted.org/six/#six.u 
why you want this change, please give me an idea.

Reply:
Python 3 compatibility is one of edX's long term goals so we should use one of `six`'s functions which provide unicode types for both Python 2 and 3

Reply:
Maybe you should use this instead, from the stackoverflow issue you linked: https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.force_text

Reply:
i changed it to `smart_text`

Reply:
Does that work for you? Looking at `smart_text` it leaves `Promise` objects as is, but don't we want to convert to string so that it's JSON serializable?

Reply:
i havent tested it yet, let me test

Reply:
force_text is not loading strings (score in given case)

<img width="922" alt="screen shot 2017-10-10 at 11 09 08 pm" src="https://user-images.githubusercontent.com/10431250/31402664-1eb355ae-ae10-11e7-95ae-29f13ed59d7e.png">

<img width="1187" alt="screen shot 2017-10-10 at 11 10 00 pm" src="https://user-images.githubusercontent.com/10431250/31402714-40348fa4-ae10-11e7-9c88-942450cdeaa3.png">